### DISCUSSION TITLE
How to exchange the Duino-Coins to USD

### DISCUSSION THREAD

**Discussion by marcoduinocoin** [revoxhere/duino-coin#756] [Show and tell]
Hi bro! Do you know how to exchange the DUCO to USD then transfer to Germany bank account in a reasonable rate?

**Comment by MuhammedRaslan** [comment]
I believe tha Node exchange would work for you.

**Reply by marcoduinocoin** [reply]
Unfortunately, the Node-S is unavailable now.

**Reply by KlasL4243** [reply]
it still is

**Comment by thealidev** [comment]
Change it using maticDUCO
